docs/topics/writing_templates.md

```diff
@@ -104,7 +104,7 @@ See [](image_tag) for full documentation.
 
 This filter takes a chunk of HTML content and renders it as safe HTML in the page. Importantly, it also expands internal shorthand references to embedded images, and links made in the Wagtail editor, into fully-baked HTML ready for display.
 
-Only fields using `RichTextField` need this applied in the template.
+You should apply this only to fields using `RichTextField
```

Comment:
```suggestion
Only fields using `RichTextField` need this applied in the template.
```

Personally I find the original text more correct. I think it’s more useful to state the need so people can infer what to do, than tell them what to do.